plugin/ui/completion.py

```diff
@@ -118,7 +121,22 @@ def _tidy_completion_index(self, index: int) -> int:
         return clamp(index, 0, completions_cnt - 1)
 
 
-class _PopupCompletion:
+class BaseCompletion(metaclass=ABCMeta):
```

Comment:
```suggestion
class BaseCompletion(ABC):
```

And `from abc import ABC`

Reply:
Sorry, forget we are at py33.